spec/models/async_transaction/base_spec.rb

```diff
@@ -55,4 +55,17 @@
       expect(record3).to be_instance_of(AsyncTransaction::Vet360::TelephoneTransaction)
     end
   end
+
+  describe '#delete_stale' do
+    it 'deletes old transactions but not new ones' do
+      create(:address_transaction,
+             created_at: (Time.current - 31.days).iso8601,
+             status: AsyncTransaction::Base::COMPLETED)
+      create(:telephone_transaction,
+             created_at: (Time.current - 29.days).iso8601,
+             status: AsyncTransaction::Base::COMPLETED)
+      AsyncTransaction::Base.new.delete_stale
+      expect(AsyncTransaction::Base.all.count).to eq(1)
```

Comment:
In addition to the count, can you add in an expectation that the correct record was saved and/or deleted?

Reply:
✅